build.sbt

```diff
@@ -17,24 +17,7 @@
 import PIOBuild._
 
 lazy val scalaSparkDepsVersion = Map(
```

Comment:
Maybe we can get rid of this `Map` as well?

Reply:
Yes, but we will add Scala 2.12 support very soon. So keeping this `Map` is better I think.